libraries/AP_RangeFinder/AP_RangeFinder_Params.cpp

```diff
@@ -129,6 +129,24 @@ const AP_Param::GroupInfo AP_RangeFinder_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ORIENT", 53, AP_RangeFinder_Params, orientation, ROTATION_PITCH_270),
 
+    // @Param: FLT_COEF
+    // @DisplayName: Coefficient for running average filter on rangefinder measurements
+    // @Description: This parameter sets the weight of the measurement moving average versus the current measurement in the filter. 0.7 is usually a good value, 0 disables
+    // @Range: 0 .95
+    // @Increment: .01
+    // @User: Advanced
+    AP_GROUPINFO("FLT_COEF", 54, AP_RangeFinder_Params, flt_coeff, 0),
```

Comment:
How about we make this a Hz instead of a coefficient?  I think they can be expressed either way and it's a straight conversion between them.. I think the AC_PID object has this conversion built in.